src/nbl/ext/ImGui/shaders/fragment.hlsl

```diff
@@ -1,9 +1,11 @@
 #include "common.hlsl"
 
+[[vk::push_constant]] struct PushConstants pc;
+
 [[vk::combinedImageSampler]][[vk::binding(0, 0)]] Texture2D sampleTexture : register(t0);
 [[vk::combinedImageSampler]][[vk::binding(0, 0)]] SamplerState linearSampler : register(s0);
```

Comment:
as soon as #693 gets merged, transition to separate descriptor bindings for the sampler and texture

Reply:
done